src/os/bluestore/BlueStore.cc

```diff
@@ -3314,11 +3328,43 @@ void BlueStore::handle_conf_change(const struct md_config_t *conf,
     throttle_bytes.reset_max(conf->bluestore_max_bytes);
     throttle_deferred_bytes.reset_max(
       conf->bluestore_max_bytes + conf->bluestore_deferred_max_bytes);
+    // reset internal throttle of the throttle manager
+    throttle_mgr.reset();
   }
   if (changed.count("bluestore_deferred_max_bytes")) {
     throttle_deferred_bytes.reset_max(
       conf->bluestore_max_bytes + conf->bluestore_deferred_max_bytes);
   }
+
+#if 0
+  // This cannot work as-is, because it will turn on throttle_ops,
```

Comment:
@liewegas Hello Sage. Its my comment that multiline comments should be inside /* */
Though these are internal not meant for customer facing docs but still 
We should maintain consistency over the code.

Reply:
@amitkumar50 OK, so suppose this is changed to /* */ What if I come along and want to comment out this whole #if/#endif, or half this file? How do I do this easily if all multi-line comments are in /**/ style? I'm not a fan of c-style comments.